src/jmcomic/jm_option.py

```
@@ -159,7 +159,7 @@ def get_rule_solver(cls, rule: str) -> Optional[RuleSolver]:
             return None
 
         def solve_func(detail):
-            return fix_windir_name(str(DetailEntity.get_dirname(detail, rule[1:])))
+            return fix_windir_name(str(DetailEntity.get_dirname(detail, rule[1:]))).strip()
```

Ruff (0.8.2)

`fix_windir_name` may be undefined, or defined from star imports

(F405)